scaffolder-templates/gitlab-workflow/template.yaml

```diff
@@ -0,0 +1,451 @@
+      action: fetch:template
+      if: ${{ parameters.CI == 'tekton_argocd' }}
+      input:
+        url: ../gitops
+        values:
+          workflowId: ${{ parameters.workflowId }}
+          namespace: ${{ parameters.namespace }}
+          argocdNamespace: ${{ parameters.argocdNamespace }}
+          orgName: ${{ parameters.orgName }}
+          repoName: ${{ parameters.repoName }}-gitops
+          owner: ${{ parameters.owner }}
+          system: ${{ parameters.system }}
+          applicationType: workflow-project
+          description: ${{ parameters.description }}
+          sourceControl: gitlab.com # CHECK if not cee redhat
+          lifecycle: development
+          gitUrl: git@${{ parameters.gitlabHost }}:${{ parameters.orgName }}/${{ parameters.repoName }}.git
+          gitOpsUrl: git@${{ parameters.gitlabHost }}:${{ parameters.orgName }}/${{ parameters.repoName }}-gitops.git
+          persistencePSQLSecretName: ${{ parameters.persistencePSQLSecretName }}
+          persistencePSQLUserKey: ${{ parameters.persistencePSQLUserKey }}
+          persistencePSQLPasswordKey: ${{ parameters.persistencePSQLPasswordKey }}
+          persistencePSQLServiceName: ${{ parameters.persistencePSQLServiceName }}
+          persistencePSQLServiceNamespace: ${{ parameters.persistencePSQLServiceNamespace }}
+          persistencePSQLServicePort: ${{ parameters.persistencePSQLServicePort }}
+          persistencePSQLDatabaseName: ${{ parameters.persistencePSQLDatabaseName }}
+          persistencePSQLDatabaseSchema: ${{ parameters.workflowId }}
+          quayOrgName: ${{ parameters.quayOrgName }}
+          quayRepoName: ${{ parameters.quayRepoName }}
+        targetPath: gitops
+    - id: renameFilesForPersistence
+      if: ${{ parameters.CI == 'tekton_argocd' }}
+      action: fs:rename
+      name: Rename files for persistence
+      input:
+        files:
+          - from: gitops/kustomize/base/sonataflow-patch-persistence.yaml
+            to: gitops/kustomize/base/sonataflow-patch.yaml
+            overwrite: true
+    - id: publishGitOps
+      if: ${{ parameters.CI == 'tekton_argocd' }}
+      name: Publishing to the GitOps Code Repository
+      action: publish:gitlab
+      input:
+        allowedHosts: ["${{ parameters.gitlabHost }}"]
+        description: Configuration repository for ${{ parameters.orgName }}/${{ parameters.repoName }}
+        repoUrl: ${{ parameters.gitlabHost }}?owner=${{ parameters.orgName }}&repo=${{ parameters.repoName }}-gitops
+        defaultBranch: main
+        sourcePath: gitops
+    - id: registerGitOps
+      if: ${{ parameters.CI == 'tekton_argocd' }}
+      name: Registering the GitOps Catalog Info Component
+      action: catalog:register
+      input:
+        repoContentsUrl: ${{ steps.publishGitOps.output.repoContentsUrl }}
+        catalogInfoPath: /catalog-info.yaml
+  output:
+    links:
+      - title: Bootstrap the GitOps Resources
+        icon: github
```

Comment:
is there a gitlab icon?

Reply:
no :( 
https://github.com/backstage/backstage/blob/master/packages/app-defaults/src/defaults/icons.tsx